SConstruct

```diff
@@ -1091,6 +1091,7 @@ if compiler == 'clang':
             '-Wno-variadic-macros',
             '-Wno-disabled-macro-expansion',
             '-Wno-c++11-long-long',
+            '-Wno-documentation-unknown-command',
```

Comment:
Why do we need it? These warnings are usually helpful.

Reply:
When I use `@INTERFACE` in the comments clang complains about unknown tag.

Reply:
Then you should try to escape or quote it.

Reply:
When I use `@INTERFACE` in the comments clang complains about unknown tag.

Reply:
`\@INTERFACE`

Reply:
Wow, it's a kind of magic :)

Reply:
Fixed